config/install/system.image.yml

```diff
@@ -0,0 +1,3 @@
+toolkit: gd
```

Comment:
Acquia recommends imagemagik: 
https://support.acquia.com/hc/en-us/articles/360005255734-Scoping-the-impact-of-image-processing#usetherightimagepackage

Explore this change.

Reply:
https://stanfordits.atlassian.net/browse/D8CORE-336